src/text_file_functions.cpp

```diff
@@ -249,6 +253,13 @@ QVector<server_type> AOApplication::read_serverlist_txt()
     f_server_list.append(f_server);
   }
 
+  server_type demo_server;
+  demo_server.ip = "127.0.0.1";
+  demo_server.port = 99999;
```

Comment:
@scatterflower the port is set to 99999 despite us checking port availability earlier, do we *really* need it to be a true TCP server anyway?